### DISCUSSION TITLE
是否可以增加一个自定义挂件的功能呢 - [一个示例的创意帖子]

### DISCUSSION THREAD

**Discussion by tangly1024** [tangly1024/NotionNext#67] [Ideas]
https://indify.co 这个网站上有很多有趣的挂件，是否可以添加到博客系统中～让用户在笔记中配置挂件地址。
<img width="955" alt="image" src="https://user-images.githubusercontent.com/15920488/156110164-c3ab6ff2-eb2c-4da8-86ac-cb3ebed8bc9f.png">

**Comment by buzhiming123** [comment]
没毛病
